src/parser/jobs/mnk/modules/RiddleOfFire.tsx

```diff
@@ -86,90 +120,122 @@ export default class RiddleOfFire extends Module {
 			this.stopAndSave()
 		}
 
-		// Aggregate GCDs under each RoF
-		const missedGcds = this.history
+		const nonRushedRiddles = this.history
 			.filter(riddle => !riddle.rushing)
-			.map(riddle => POSSIBLE_GCDS - this.getGcdCount(riddle))
-			.reduce((total, current) => total + current, 0)
+
+		// This could be redundant with GCDs spent not under Wind, but according to Tiff you should only
+		// be in FoF during a rushed Riddle, so I'm not too worried about it.
+		const droppedGcds = (nonRushedRiddles.length * EXPECTED_GCDS - 1) // opener Riddle has 10 GCDs
+			- nonRushedRiddles.reduce((sum, riddle) => sum + riddle.gcds, 0)
+
+		const droppedElixirFields = (nonRushedRiddles.length) // should be 1 per Riddle
+			- nonRushedRiddles.reduce((sum, riddle) => sum + riddle.elixirField, 0)
+
+		// Keep these seperate for different suggestions; our baseline is 1 charge, but the MNK mentors
+		// want a minor suggestion to track Riddles that only had 1 Tackle
+		const riddlesWithOneTackle = nonRushedRiddles.filter(riddle => riddle.tackles === 1).length
+		const riddlesWithZeroTackles = nonRushedRiddles.filter(riddle => riddle.tackles === 0).length
+		const droppedExpectedOgcds = droppedElixirFields + riddlesWithZeroTackles
 
 		this.suggestions.add(new TieredSuggestion({
 			icon: ACTIONS.RIDDLE_OF_FIRE.icon,
 			content: <Trans id="mnk.rof.suggestions.gcd.content">
-				Aim to hit {POSSIBLE_GCDS} GCDs into each <StatusLink {...STATUSES.RIDDLE_OF_FIRE} />.
+				Aim to hit {EXPECTED_GCDS} GCDs during each <StatusLink {...STATUSES.RIDDLE_OF_FIRE} />.
```

Comment:
mediumish; I feel these could probably do with a bit of expanding with the content, they're a bit terse and repetitive.